lua/indent_blankline/utils.lua

```diff
@@ -187,26 +187,25 @@ M.find_indent = function(whitespace, only_whitespace, shiftwidth, strict_tabs, l
 end
 
 M.get_current_context = function(type_patterns, use_treesitter_scope)
-    local ts_utils_status, ts_utils = pcall(require, "nvim-treesitter.ts_utils")
-    if not ts_utils_status then
-        vim.schedule_wrap(function()
-            M.error_handler("nvim-treesitter not found. Context will not work", vim.log.levels.WARN)
-        end)()
-        return false
-    end
-    local locals = require "nvim-treesitter.locals"
     local cursor_node = vim.treesitter.get_node()
 
     if use_treesitter_scope then
-        local current_scope = locals.containing_scope(cursor_node, 0)
-        if not current_scope then
-            return false
-        end
-        local node_start, _, node_end, _ = current_scope:range()
-        if node_start == node_end then
-            return false
+        local success, locals = pcall(require, "nvim-treesitter.locals")
+        if not success then
+            vim.schedule_wrap(function()
+                M.error_handler("nvim-treesitter not found. Defaulting to patterns.", vim.log.levels.WARN)
```

Comment:
Whoops. I didn't realize this change would be automatically pushed to the PR as well. I reverted and changed the check to `nvim_treesitter.locals` since `nvim_treesitter.ts_utils` is not used anymore.